index.html

```diff
@@ -887,7 +887,7 @@ <h4>parseDerivedProofValue</h4>
 The following algorithm parses the components of the derived proof value.
 The required input is a derived proof value (<var>proofValue</var>). A
 A single <em>derived proof value</em> value object is produced as output, which
-contains a set of five elements, using the names "bbsProof", "labelMap",
+contains a set of four elements, using the names "bbsProof", "labelMap",
 "mandatoryIndexes", and "selectiveIndexes".
```

Comment:
Unrelated to the count, should this use the same styling as "names" elsewhere?
```suggestion
contains a set of four elements, using the names "`bbsProof`", "`labelMap`",
"`mandatoryIndexes`", and "`selectiveIndexes`".
```